docs/language-server/protocol-language-server.md

```diff
@@ -2235,8 +2176,7 @@ addition of the root in order to inform them of the content root's ID.
 
 ```typescript
 {
-  id: UUID; // The content root ID
-  absolutePath: [String];
+  root: ContentRoot;
```

Comment:
Just a reminder to make sure that  the appropriate task is scheduled on the IDE side

Reply:
@farmaazon @MichaelMauderer I understand that this will likely need to be integrated for the https://github.com/enso-org/ide/issues/1634
Besides, I understand that before that the `file/rootAdded` notification was not being handled. Is it a breaking change for the IDE that we will start sending it? Will it cause errrors if you have no handler for it? If yes, I'd like to ask if you can add at least a dummy handler for compatibility which can later be extended to its full functionality.